packages/react-interactions/events/src/dom/__tests__/PressLegacy-test.internal.js

```diff
@@ -914,8 +914,7 @@ describe.each(environmentTable)('Press responder', hasPointerEvents => {
       ReactDOM.render(<Component />, container);
 
       const target = createEventTarget(ref.current);
-      target.keydown({key: 'Enter'});
-      target.click({preventDefault});
```

Comment:
Why was the `click` event removed from this test? When Enter is pressed is certain host elements it will cause a `click` to be dispatched

Reply:
Not if you `preventDefault`, meaning `click` will never fire.

Reply:
The reason we preventDefault on Spacebar it because otherwise it scrolls the window. Should we preventDefault on keydown for Enter though?

Reply:
I think so, unless you know of any reason why we should not?